roles/slurm_manager/tasks/main.yml

```diff
@@ -63,13 +63,23 @@
     - "{{ dev_tools }}"
   tags: install
 
+- name: Verify if slurm is installed
```

Comment:
Would it make more sense to use `package_facts` to gather this information and store it in the ansible facts so that we do not have to make the query in each separate playbook?

Reply:
Since we are verifying only slurm RPMs , grep is used to gather slurm RPMs. Also package_facts gathers information of all packages and RPMs installed in the node. To simplify this, have used grep command.

Reply:
I think this will work for now, but going forward we may want to test for the existence of other packages, such as accelerator drivers (see #231).

Reply:
@lwilson sure. we will take this into account for our upcoming development work.